src/SearchBar.js

```diff
@@ -2,23 +2,26 @@ import React, {Component} from 'react';
 import './SearchBar.css';
 
 export default class SearchBar extends Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = {
-
+      text: ''
     }
   }
-  handleChange = event => {
-    this.setState({SOMETHING: 0})
-  }
-  searchTitles() {
-
+  updateText = event => {
+    let searchText = event.target.value.toLowerCase()
```

Comment:
SearchBar.js line 12 + line 18, grabbing the values and changing it to lower case will prevent any weird search issues that might have occurred otherwise, good thinking